src/Helper/RoleHelper.js

```diff
@@ -0,0 +1,21 @@
+const config = require('config');
+
+module.exports = {
+    hasRoles: function (member) {
```

Comment:
```suggestion
    hasRole: function (member, role) {
        return this.hasRoles(member) && member.roles.cache.has(role);
    },
    hasRoles: function (member) {
```

Reply:
hasRole is already a part of the Discord.js implementation, there would be little use to implement it like this. This function in particular checks if a user has any role assigned to it.

Reply:
Additionally, the reason for the muted / administrator functions is because those are from config. In the case of administrator, it's a collection of possible administrator roles.